### DISCUSSION TITLE
Make option to choose a default domain for the Alias Domain

### DISCUSSION THREAD

**Discussion by sientelo** [anonaddy/anonaddy#496] [Feature Requests]
Would be even better if one could also choose a default Alias format :)

I always want to use my custom domain and 'random words' for the alias format

![2023-08-16_19-52](https://github.com/anonaddy/anonaddy/assets/35114600/607bda7e-9f1f-4053-a3bd-605be7e6551b)

![2023-08-16_19-55](https://github.com/anonaddy/anonaddy/assets/35114600/49bbd072-e312-4581-9353-418fa1234562)

Regardless, I will always love Addy :100:

**Comment by willbrowningme** [comment]
Thanks, this is already possible to do from the settings page - [https://app.addy.io/settings](https://app.addy.io/settings) under:

"Update Default Alias Domain" and "Update Default Alias Format".

**Reply by sientelo** [reply]
sorry! I swear I searched before posting. Thanks Will
